ignite/handlers/checkpoint.py

```diff
@@ -664,21 +665,26 @@ class DiskSaver(BaseSaveHandler):
     """
 
     def __init__(
-        self, dirname: str, atomic: bool = True, create_dir: bool = True, require_empty: bool = True, **kwargs: Any
+        self,
+        dirname: Union[str, Path],
+        atomic: bool = True,
+        create_dir: bool = True,
+        require_empty: bool = True,
+        **kwargs: Any,
     ):
         self.dirname = os.path.expanduser(dirname)
```

Comment:
Let's make self.dirname as Path object. Can you please find an equivalent to `os.path.expanduser(dirname)`, it should do `~/folder` -> `/home/$USER/folder`

Reply:
OK, I understand.
I will use `Path.expanduser()`.